package.json

```diff
@@ -3,7 +3,8 @@
   "version": "7.0.2",
   "description": "Types used in the FiatConnect specification. Offered as standalone module for payment providers and wallets to both use for FiatConnect APIs and integrations.",
   "scripts": {
-    "prepublish": "tsc",
+    "typecheck": "tsc",
+    "prepublish": "yarn typecheck",
```

Comment:
should we clean dist before running this? I don't think this is a problem yet for this repo, but when I built it locally when renaming some files, not cleaning dist published an outdated file. Although I guess if we do publishes with GHA, we should always have a clean dist.